Task3/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int A = Convert.ToInt32(Console.ReadLine());
```

Comment:
Локальные переменные в C# принято называть с маленькой буквы
Методы класса `Convert` лучше не использовать без особой необходимости. Используйте методы `Parse()` или `TryParse()`
```suggestion
            int a = int.Parse(Console.ReadLine());
```